guide/ops/locations/_more-clouds.md

```diff
@@ -300,7 +306,94 @@ Configuration of floating IPs is as networks; specify the pools to use as anothe
           - "pool name"
 
 
+### Basic Location Structure
+
+This is a basic inline YAML template for an OpenStack location:
+
+```
+location:
+  jclouds:clouds:openstack-nova:
+    endpoint: http://x.x.x.x:5000/v2.0/
+    identity: "your-tenant:your-username"
+    credential: your-password
+
+    imageId: your-region-name/your-image-id
+    hardwareId: your-region-name/your-flavor-id
+    loginUser: 'ubuntu'
+    loginUser.privateKeyFile: /path/to/your/privatekey
+    jclouds.openstack-nova.auto-generate-keypairs: false
+    jclouds.openstack-nova.auto-create-floating-ips: true
+
+    templateOptions:
+      networks: [ "your-network-id" ]
+      floatingIpPoolNames: [ "your-floatingIp-pool" ]
+      securityGroups: ['your-security-group']
+      keyPairName: "your-keypair"
```

Comment:
with `jclouds.openstack-nova.auto-generate-keypairs: false` then you may want to specify a `keyPairName` otherwise keyPairName is optional. We should find a better way to make it explicit